examples/fast-build/package.json

```diff
@@ -4,9 +4,9 @@
   "private": true,
   "scripts": {
     "dev": "astro dev --experimental-static-build",
-    "start": "astro dev",
+    "start": "astro dev --experimental-static-build",
     "build": "astro build --experimental-static-build",
-    "scan-build": "astro build",
+    "scan-build": "astro build --experimental-static-build",
```

Comment:
this is called "scan-build" because I wanted to test the old legacy build vs. the static build when it was first being developed. Can probably just drop this script now, it's not needed.

Reply:
ah, apologies! I'll remove for now